pywinauto/base_wrapper.py

```diff
@@ -39,79 +40,17 @@ class ElementNotVisible(RuntimeError):
     pass
 
 #=========================================================================
-class _MetaWrapper(type):
-    "Metaclass for Wrapper objects"
-    re_wrappers = {}
-    str_wrappers = {}
-    control_types = {}
-
-    def __init__(cls, name, bases, attrs):
-        # register the class names, both the regular expression
-        # or the classes directly
-
-        #print("metaclass __init__", cls)
-        type.__init__(cls, name, bases, attrs)
-
-        for win_class in cls.windowclasses:
-            _MetaWrapper.re_wrappers[re.compile(win_class)] = cls
-            _MetaWrapper.str_wrappers[win_class] = cls
-        for control_type in cls.controltypes:
-            _MetaWrapper.control_types[control_type] = cls
-
-    @staticmethod
-    def FindWrapper(element):
-        "Find the correct wrapper for this native element"
-        if isinstance(element, six.integer_types):
-            from ..NativeElementInfo import NativeElementInfo
-            element = NativeElementInfo(element)
-        class_name = element.className
-
-        try:
-            return _MetaWrapper.str_wrappers[class_name]
-        except KeyError:
-            wrapper_match = None
-
-            for regex, wrapper in _MetaWrapper.re_wrappers.items():
-                if regex.match(class_name):
-                    wrapper_match = wrapper
-                    _MetaWrapper.str_wrappers[class_name] = wrapper
-
-                    return wrapper
-
-        # if it is a dialog then override the wrapper we found
-        # and make it a DialogWrapper
-        if handleprops.is_toplevel_window(element.handle):
-            from . import win32_controls
-            wrapper_match = win32_controls.DialogWrapper
-
-        if wrapper_match is None:
-            from .HwndWrapper import HwndWrapper
-            wrapper_match = HwndWrapper
-        return wrapper_match
-
+@six.add_metaclass(abc.ABCMeta)
+class BaseMeta(abc.ABCMeta):
+    "Abstract metaclass for Wrapper objects"
+    
     @staticmethod
```

Comment:
It should work in Python3.x but in practice it doesn't work in both Py2 and Py3. I can instantiate `BaseMeta` object despite it's "abstract". See [SO thread](http://stackoverflow.com/questions/4474395/staticmethod-and-abc-abstractmethod-will-it-blend) about that. So I decided to avoid this obsolete construction because I don't believe it's really useful.